libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationCommon.java

```diff
@@ -97,19 +98,27 @@ public Notification createNotification(final String recipient,
     }
 
     void addDomainRoleRecipients(Notification notification, final String domainName, final String roleName) {
-        Set<String> domainRoleMembers = domainRoleMembersFetcher.getDomainRoleMembers(domainName, roleName);
-        if (domainRoleMembers == null || domainRoleMembers.isEmpty()) {
+        try {
```

Comment:
This try catch is not really necessary now that we handle groups as role members. However, it's best to catch any exception thrown for a specific recipient so the notification will still be sent to other recipients.